web/src/models/Test.model.ts

```diff
@@ -9,13 +9,15 @@ const Test = ({
   specs,
   version = 1,
   serviceUnderTest: rawTrigger,
+  summary = {},
 }: TRawTest): TTest => ({
   id,
   name,
   version,
   description,
   definition: TestSpecs(specs || {}),
   trigger: Trigger(rawTrigger || {}),
+  summary,
```

Comment:
Do we need to create a model for the new info? or do you think we should be OK with how it is?

Reply:
Created a model. Thanks for the suggestion.